### DISCUSSION TITLE
Old Esri Vector Basemaps will be retired in December 2022

### DISCUSSION THREAD

**Discussion by mbranscomb** [NASA-LIS/LISF#1238] [General]
Hi,

This information may be relevant to the NASA Land Information System Framework (LISF) project:

The following versions of Esri Vector Basemaps will be retired in December 2022 and will no longer be available to use in maps and apps:
- World_Basemap (version 1) https://basemaps.arcgis.com/v1/arcgis/rest/services/World_Basemap/VectorTileServer
- World Basemap WGS84 (version 1) https://basemaps.arcgis.com/v1/arcgis/rest/services/World_Basemap_WGS84/VectorTileServer

The current versions, identified as `version 2` or `v2`, were first released in December 2017. If you started using Esri vector basemaps after this date, you likely are using a v2 basemap and don’t need to make any changes. If you’re unsure, keep reading.

- If you have an ArcGIS Developer account/subscription, you should refer to [basemaps for ArcGIS developers](https://developers.arcgis.com/documentation/mapping-apis-and-services/maps/services/basemap-layer-service/).
- If you have an ArcGIS Online account or ArcGIS Enterprise account, see [ArcGIS Online Basemaps](https://www.arcgis.com/home/group.html?id=702026e41f6641fb85da88efe79dc166#overview).

Thanks

**Comment by dmocko** [comment]
Hi @mbranscomb - Thanks for letting us know.
